examples/lab-robust/v2/qwiklabs.yaml

```diff
@@ -105,12 +173,21 @@ environment:
   - label: GCP Folder Display Name
     reference: primary_folder.display_name
 
+instruction:
+  type: html
```

Comment:
That's being developed by @julianaviola atm.

Reply:
+1 i.e. I would not include `html` right now.